### PR TITLE
0.x

### PR DESCRIPTION
**Pull request by bFekete**
Adds the ability for Puppet to manage: virtual hosts, host aliases, cookies, and custom services.

Also fixed some of the JVM parameters.

### CODE REVIEW

**Comment by bFekete** [comment]
@bmjen Could you look over the pull request?

**Comment by bFekete** [comment]
@bmjen Could you look over the pull request?

File: lib/puppet/provider/websphere_host_alias/wsadmin.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+require 'puppet/provider/websphere_helper'
+
+Puppet::Type.type(:websphere_host_alias).provide(:wsadmin, :parent => Puppet::Provider::Websphere_Helper) do
+
+  def exists?
+    # If it doesn't exist, the wsadmin should fail but since :failonfail is false it will create 
+    ## (J|P)ython is whitespace sensitive, and this bit doesn't do well when
+    ## being passed as a normal command-line argument. 
+    cmd = <<-END
+host_aliases =  AdminConfig.list('HostAlias', AdminConfig.getid( '/Cell:CELL_01/VirtualHost:test_vhost/')).split(lineSeparator)
```
Comment (joshbeard) [inline lib/puppet/provider/websphere_host_alias/wsadmin.rb]: @bFekete should the cell and virtualhost name be hard coded here?

File: lib/puppet/provider/websphere_host_alias/wsadmin.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    cmd += "[port \"#{resource[:port]}\"]]')\""
+    
+    self.debug "Running #{cmd}"
+    result = wsadmin(:command => cmd, :user => resource[:user])
+    self.debug result
+  end
+  
+  def destroy
+    cmd = <<-END
+host_aliases =  AdminConfig.list('HostAlias', AdminConfig.getid( '/Cell:CELL_01/VirtualHost:test_vhost/')).split(lineSeparator)
```
Comment (joshbeard) [inline lib/puppet/provider/websphere_host_alias/wsadmin.rb]: @bFekete should the cell and virtualhost name be hard coded here?

**Comment by bFekete** [comment]
@Josh beard No, I'll fix it when I get a chance. Thanks for looking at it.

**Pull request closed without merging**